### PR TITLE
fix: Exclude Vite specific import parameters for project's file imports

### PR DESCRIPTION
**Pull request by mshabarov**
Excludes parameters added by Vite (e.g. ?inline) for frontendFiles entries in the stats.json of dev-bundle.
These parameters cause a mistake in import string comparison and false re-bundling.

Related-to https://github.com/vaadin/flow/issues/16109
Follow-up for https://github.com/vaadin/flow/pull/16039

### CODE REVIEW

File: flow-server/src/main/resources/vite.generated.ts [hunk 1]
```diff
@@ -225,7 +225,8 @@ function statsExtracterPlugin(): PluginOption {
       const generatedImports = readFileSync(path.resolve(generatedFlowImportsFolder, "generated-flow-imports.js"), {encoding: 'utf-8'})
           .split("\n")
           .filter((line: string) => line.startsWith("import"))
-          .map((line: string) => line.substring(line.indexOf("'")+1, line.lastIndexOf("'")));
+          .map((line: string) => line.substring(line.indexOf("'")+1, line.lastIndexOf("'")))
```
Comment (czp13) [inline flow-server/src/main/resources/vite.generated.ts:228]: smallie :), but this line could be reformated like:
```suggestion
          .map((line: string) => line.substring(line.indexOf("'") + 1, line.lastIndexOf("'")))
```

**Review by czp13 — COMMENTED** [review]
As I see we have new tests for the CSS bundle part, but for the other changes about the modules how do we handle the `?` in the stream (was refactored and slightly changed to happen in one step before then it happened). I do not see tests (maybe I missed them):
`flow-server/src/main/resources/vite.generated.ts`. 

If it is possible to add, then maybe would be good to cover the changes that happened in the `vite.generated.ts`.

File: flow-server/src/main/resources/vite.generated.ts (hunk 1 above)
Comment (mshabarov) [inline flow-server/src/main/resources/vite.generated.ts:228]:
```suggestion
          .map((line: string) => line.substring(line.indexOf("'") + 1, line.lastIndexOf("'")))
```

**Review by czp13 — APPROVED** [review]
Thank you very much for covering my question. It is making sense, we do not win big with this refactor, mock, but can lose a lot of time, I agree! I am approving from my side.

**Pull request merged**